demo/README.md

```diff
@@ -0,0 +1,203 @@
+# in-toto demo
+
+In this demo, we will use in-toto to secure a supply chain with a very simple
+workflow.
+Alice will be the project owner - she creates and signs the supply chain layout
+with her private key - and Bob and Carl will be the project functionaries - they
+carry out the steps of the supply chain as defined in the layout.
+
+For the sake of demonstrating in-toto, we will have you run all parts of the
+supply chain.
+This is, you will perform the commands on behalf of Alice, Bob and Carl as well
+as the client who verifies the final product.
+
+
+### Download and setup in-toto on *NIX (Linux, OS X, ..)
+```shell
+# Make sure you have git, python and pip installed on your system
+# and get in-toto
+git clone -b develop --recursive https://github.com/in-toto/in-toto.git
+
+# Change into project root directory
+cd in-toto
+
+# Install required packages (we strongly recommend using Virtual Environments)
+# http://docs.python-guide.org/en/latest/dev/virtualenvs/
+pip install -r requirements.txt
+
+# Install additional requirements that for some good reason are not in the
+# requirements file
+pip install pycrypto, cryptography
```

Comment:
You'd get an error if this command were executed.  I think the comma shouldn't be there.